web/explorer/src/composables/useRdocLoader.js

```diff
@@ -0,0 +1,89 @@
+// useDataLoader.js
+import { ref, readonly } from "vue";
+import { useToast } from "primevue/usetoast";
+
+export function useRdocLoader() {
+    const toast = useToast();
+    const rdocData = ref(null);
+    const isValidVersion = ref(false);
+
+    const MIN_SUPPORTED_VERSION = "7.0.0";
+
+    /**
+     * Checks if the loaded rdoc version is supported
+     * @param {Object} rdoc - The loaded JSON rdoc data
+     * @returns {boolean} - True if version is supported, false otherwise
+     */
+    const checkVersion = (rdoc) => {
+        const version = rdoc.meta.version;
+        if (version < MIN_SUPPORTED_VERSION) {
+            console.error(
+                `Version ${version} is not supported. Please use version ${MIN_SUPPORTED_VERSION} or higher.`
+            );
+            toast.add({
+                severity: "error",
+                summary: "Unsupported Version",
+                detail: `Version ${version} is not supported. Please use version ${MIN_SUPPORTED_VERSION} or higher.`,
+                life: 5000,
+                group: "bc" // bottom-center
```

Comment:
maybe we want to show this in the center-center?

Reply:
I think this will hide the main text form the user, and I don't believe this placement is supported by default.